packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -40,5 +43,18 @@
   },
   "nyc": {
     "reporter": ["text-summary", "lcov"]
+  },
+  "contributes": {
+    "configuration": {
+      "title": "Salesforce DX Gating",
+      "properties": {
+        "sfdx.refreshSObjects.enabled": {
```

Comment:
The way it's written, I believe that this setting will be publicly visible when people go to Code > Settings. That's intentional? I think that should be fine since this feature is going to be stable enough soon and this setting can be removed.

There is also the way of doing a hidden setting. This is used by the tslint VS code extension for turning on extra logging. See https://github.com/Microsoft/vscode-tslint/blob/9477a7e48e0ed4029923aa51ebf7953ca0cfd8ea/tslint/README.md#faq

Reply:
I will do hidden - I didn't quite realize how to do it - thanks

Reply:
I see tslint.trace.server - doesn't seem to be hidden.  And I tried the same scope: "window" that they used and I still see the setting..      I think I won't worry about it for now - it's temporary in any case.    If you see what I missed to make it hidden, let me know and I'll change it.

Reply:
![screen shot 2017-09-21 at 8 45 49 pm](https://user-images.githubusercontent.com/54414/30728591-e7904c00-9f0d-11e7-8d20-94b56789c8db.png)

Yes, you're right. Apologies for leading you down a rabbit hole.